adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -128,10 +128,10 @@ private static long ConvertToTimeT(DateTime time)
             return (long)(diff.TotalSeconds);
         }
 
-        private string Encode(IClientAssertionCertificate credential)
+        private string Encode(IClientAssertionCertificate credential, bool sendX5C)
```

Comment:
sendX5C name does not reflect usage of parameter

Reply:
@trwalke

---
In reply to: [202846635](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r202846635) [](ancestors = 202846635)

Reply:
Updated parameter to "sendCertificate"